src/v/config/configuration.cc

```diff
@@ -436,6 +436,26 @@ configuration::configuration()
       "truncation if any of the replicas has more data than the majority.",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       true)
+  , raft_max_inflight_follower_append_entries_requests(
+      *this,
+      "raft_max_inflight_follower_append_entries_requests",
+      "Maximum number of append entries requests, that were sent to a node "
+      "from a seastar shard, and that the current node is waiting for to be "
+      "replied. (previously this number was configured per raft group, now it "
+      "is configured per shard)",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      1024,
+      {.min = 1, .max = 50000})
+  , raft_max_buffered_follower_append_entries_bytes(
+      *this,
+      "raft_max_buffered_follower_append_entries_bytes",
+      "Total size of requests that may be cached per shard in Raft buffered "
```

Comment:
Suggestion for changes to description. @mmaslankaprv, please check for accuracy:

The total size of append entry requests that may be cached per shard, using Raft buffered protocol. When an entry is cached the leader can continue serving requests because the ordering of the cached requests cannot change.

Reply:
nit: I think we should also callout that particular configuration is blocking from a caller perspective if breached and propagates back pressure.